pkg/grpc/databroker/syncer.go

```diff
@@ -148,6 +155,12 @@ func (syncer *Syncer) sync(ctx context.Context) error {
 			return err
 		}
 
+		syncer.log().Info().
```

Comment:
I think this should be `Debug`?

Reply:
I used the same log level as i.e. 
```log
{"level":"info","service":"databroker","peer":"127.0.0.1:53995","type":"type.googleapis.com/pomerium.config.Config","id":"dashboard","time":"2021-04-06T15:13:46-04:00","message":"get"}
```

Reply:
Well its going to be a lot of log lines. In our case ~40k?

Reply:
I agree this is a better fit for debug. 

And in this case, we may want to even hide this under the verbose flag. 

And we can follow up with another PR to see that databroker set / gets are also debug.